mappings/AdReEngagementState.mapping

```diff
@@ -0,0 +1 @@
+CLASS rbe AdReEngagementState
```

Comment:
I'd make the `E` lowercase for readability, `to reengage` is also a correct spelling